### DISCUSSION TITLE
Compatibility with POWMR Wifi-VM dongle

### DISCUSSION THREAD

**Discussion by froias16** [odya/esphome-powmr-hybrid-inverter#41] [Q&A]
Hi will this integration work with POWMR's Wifi-VM dongle? Thanks.

**Comment by sakhnik** [comment]
I wonder too. So far, I haven't been able to get them to communicate. I figured out that modbus should be configured with `flow_control_pin: 4`. But the inverter keeps resetting the dongle every ten seconds.

**Reply by aver-ua** [reply]
Take a look at this project: https://github.com/alexeyden/openess
